### PR TITLE
revert decamelize and humanize-string to old versions

### PR DESCRIPTION
**Pull request by chris48s**
closes https://github.com/badges/shields/issues/7447
closes https://github.com/badges/shields/issues/7324

### CODE REVIEW

**Review by calebcartwright — DISMISSED** [review]
Sounds like this may not address some specific issues, but given the linked upstream issue it seems like we may want to downgrade this regardless so providing an approval accordingly

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "cloudflare-middleware": "^1.0.4",
     "config": "^3.3.6",
     "cross-env": "^7.0.3",
-    "decamelize": "^6.0.0",
+    "decamelize": "^5.0.1",
```
Comment (LitoMore) [inline package.json:36]: You should pin it at [version 3](https://github.com/sindresorhus/decamelize/releases/tag/v4.0.0). The `humanize-string` needs to upgraded as well.

```diff
# dependencies
-    "decamelize": "^6.0.0",
+    "decamelinze": "3.2.0",

# devDependencies
- "humanize-string": "^3.0.0",
+ "humanize-string": "2.0.0",
```

**Review by LitoMore — CHANGES_REQUESTED** [review]

**Comment by LitoMore** [comment]
The title of the PR needs to be updated.

**Review by LitoMore — CHANGES_REQUESTED** [review]
Let's remove the `^` mark to make sure dependabot will not update these dependencies by accident.

And please run `npm ci` after these suggestions are applied to ensure that the lockfile is up to date.

**Comment by chris48s** [comment]
Dependabot bumps dependencies even if they're pinned in package.json e.g: https://github.com/badges/shields/pull/7431/files
Assuming we get a confirmation on https://github.com/badges/shields/issues/7447 I'll add entries to https://github.com/badges/shields/blob/4fef1335d238a4f89a778e857d265b43273dd184/.github/dependabot.yml#L11 to tell dependabot to ignore them

**Review by calebcartwright — APPROVED** [review]
:heart:

**Comment by calebcartwright** [comment]
Let's go ahead and merge+deploy

**Pull request merged**